api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -666,9 +688,156 @@ public void shortenedStackTrace_shouldReturnNullIfStackTraceIsNull() {
+		List<Concept> helperList = OpenmrsUtil.conceptListHelper("name:YES|name:NO|name:DIED|set:25|set:26");
+		List<Concept> serviceList = new ArrayList<>();
+		Concept concept_auxiliar;
+		//Adding the concepts in the list to compare with the first one
+		serviceList.add(cs.getConceptByName("YES"));
+		serviceList.add(cs.getConceptByName("NO"));
+		serviceList.add(cs.getConceptByName("DIED"));
+		
+		concept_auxiliar = cs.getConcept(25);
+		serviceList.addAll(cs.getConceptsByConceptSet(concept_auxiliar));
+		
+		concept_auxiliar = cs.getConcept(26);
+		serviceList.addAll(cs.getConceptsByConceptSet(concept_auxiliar));
+		
+		assertTrue("List should be same size to be equal",helperList.size() == serviceList.size());
+		//Removing common elements
+		Set<Concept> serviceSet = new HashSet<>(serviceList);
+		helperList.stream()
+				.forEach(concept -> {
+					serviceSet.remove(concept);
+				});
+		
+		assertTrue("The set should be empty" ,serviceSet.isEmpty());
+	}
+	
+	/**
+	 * @see OpenmrsUtil#getFileAsString(File)
+	 */
+	@Test
+	public void getFileAsString_shouldReturnTheStringContainedInFile() {
+		try {
+			File file = createFileInUserDirectory("file");
+			file.createNewFile();
+			PrintWriter writer = new PrintWriter(file);
+			String output = "The first line" + "\n" + "The second line";
+			writer.println("The first line");
+			writer.print("The second line");
+			writer.close();
+			assertTrue(output.equals(OpenmrsUtil.getFileAsString(file)));
+			file.delete();
+			
+		}
+		catch (IOException e) {}
+		
+	}
+
+	/**
+	 * @see OpenmrsUtil#conceptSetHelper(String) 
+	 */
+	@Test
+	public void conceptSetHelper_shouldReturnSetOfConceptsWithGivenNameInDescriptor() {
+		Set<Concept> helperSet = OpenmrsUtil.conceptSetHelper("name:YES|name:NO|name:DIED|set:25|set:26");
+		Set<Concept> serviceSet = new HashSet<>();
+		Concept concept_auxiliar;
+		//Adding the concepts in the list to compare with the first one
+		serviceSet.add(cs.getConceptByName("YES"));
+		serviceSet.add(cs.getConceptByName("NO"));
+		serviceSet.add(cs.getConceptByName("DIED"));
+
+		concept_auxiliar = cs.getConcept(25);
+		serviceSet.addAll(cs.getConceptsByConceptSet(concept_auxiliar));
+
+		concept_auxiliar = cs.getConcept(26);
+		serviceSet.addAll(cs.getConceptsByConceptSet(concept_auxiliar));
+
+		assertTrue("List should be same size to be equal",helperSet.size() == serviceSet.size());
+		//Removing common elements
+
+		helperSet.stream()
+			.forEach(concept -> {
+				serviceSet.remove(concept);
+			});
+
+		assertTrue("The set should be empty" ,serviceSet.isEmpty());
+	}
+
+	/**
+	 * @see OpenmrsUtil#conceptListHelper(String)
+	 */
+	@Test
+	public void conceptSetHelper_shouldReturnEmptyListWhenGivenDescriptorIsNullOrEmpty() {
+		assertTrue(OpenmrsUtil.conceptSetHelper("").isEmpty());
+		assertTrue(OpenmrsUtil.conceptSetHelper(null).isEmpty());
+	}
+	
+		/**
+         * @see OpenmrsUtil#nullSafeEqualsIgnoreCase(String,String)
+         */
```

Comment:
I think it would look neater if  @see at line 837 started below the ending curly brace of conceptSetHelper_shouldReturnEmptyListWhenGivenDescriptorIsNullOrEmpty() :)